package.json

```diff
@@ -45,10 +47,13 @@
     "@typescript-eslint/eslint-plugin": "^5.21.0",
     "@typescript-eslint/parser": "^5.21.0",
     "assert": "^2.0.0",
+    "axios": "^0.27.2",
+    "dotenv": "^16.0.1",
```

Comment:
This dependency is not alphabetically ordered. I was trying to find because dotenv is already used in some packages, but I saw dotenv-cli some lines below

Reply:
done!